spark/src/main/scala/geotrellis/spark/summary/polygonal/PolygonalSummaryTileLayerCollectionMethods.scala

```diff
@@ -1,175 +0,0 @@
-/*
- * Copyright 2016 Azavea
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package geotrellis.spark.summary.polygonal
-
-import geotrellis.raster._
-import geotrellis.raster.summary.polygonal._
-import geotrellis.tiling._
-import geotrellis.spark._
-import geotrellis.util._
-import geotrellis.vector._
-import org.apache.spark.Partitioner
-import org.apache.spark.rdd._
-
-import scala.reflect.ClassTag
-
-abstract class PolygonalSummaryTileLayerCollectionMethods[
```

Comment:
Just realized these collection versions just got hit in the purge.  locationtech/geotrellis#3004 should handle developing a plan for reconstructing these these so I'll just leave them deleted for now.